admin/modules/config/settings.php

```diff
@@ -958,18 +958,51 @@
 			}
 		}
 
-		// Have we opted for a reCAPTCHA or hCaptcha and not set a public/private key?
-		if((isset($mybb->input['upsetting']['captchaimage']) && in_array($mybb->input['upsetting']['captchaimage'], array(4, 5)) && (!$mybb->input['upsetting']['recaptchaprivatekey'] || !$mybb->input['upsetting']['recaptchapublickey']))
-		   || (in_array($mybb->settings['captchaimage'], array(4, 5)) && (!$mybb->settings['recaptchaprivatekey'] || !$mybb->settings['recaptchapublickey']))
-		   || (isset($mybb->input['upsetting']['captchaimage']) && in_array($mybb->input['upsetting']['captchaimage'], array(8)) && (!$mybb->input['upsetting']['recaptchaprivatekey'] || !$mybb->input['upsetting']['recaptchapublickey'] || !$mybb->input['upsetting']['recaptchascore']))
-		   || (in_array($mybb->settings['captchaimage'], array(8)) && (!$mybb->settings['recaptchaprivatekey'] || !$mybb->settings['recaptchapublickey'] || !$mybb->settings['recaptchascore']))
-		   || (isset($mybb->input['upsetting']['captchaimage']) && in_array($mybb->input['upsetting']['captchaimage'], array(6, 7)) && (!$mybb->input['upsetting']['hcaptchaprivatekey'] || !$mybb->input['upsetting']['hcaptchapublickey']))
-		   || (in_array($mybb->settings['captchaimage'], array(6, 7)) && (!$mybb->settings['hcaptchaprivatekey'] || !$mybb->settings['hcaptchapublickey'])))
-		{
-			$mybb->input['upsetting']['captchaimage'] = 1;
+		require_once MYBB_ROOT.'inc/class_captcha.php';
+		
+		// Have we opted for a reCAPTCHA or hCaptcha and not set a public/private key in input?
+		$set_captcha_image = false;
+		if(isset($mybb->input['upsetting']['captchaimage'])){
```

Comment:
Sorry for the bureaucracy but the 1.8 development standards require the opening brace on the next line, here and elsewhere in the PR.

https://docs.mybb.com/1.8/development/standards/#control-structures